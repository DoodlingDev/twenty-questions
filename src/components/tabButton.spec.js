/* eslint require-jsdoc: "off" */
import React from "react";
import TabButton from "./tabButton";
import /* shallow,  mount */ "enzyme";
import renderer from "react-test-renderer";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

describe("rendering", () => {
  it("renders array types correctly", () => {
    const render = renderer.create(<TabButton />).toJSON();
    expect(render).toMatchSnapshot();
  });
});
