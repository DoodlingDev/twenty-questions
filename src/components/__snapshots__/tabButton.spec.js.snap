// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering renders array types correctly 1`] = `
<button
  className="q20_tabbedForm__button"
  onClick={undefined}
/>
`;
